python/whylogs/core/schema.py

```diff
@@ -123,6 +126,7 @@ def _resolve_pdf(self, df: pd.DataFrame, force_resolve: bool = False) -> bool:
             col_dtype = df.dtypes[col_name]
             self._columns[col_name] = ColumnSchema(
                 dtype=col_dtype,
+                cfg=self.default_configs,
```

Comment:
We need to pass the frozen MetricConfig to the ColumnSchema so that config will apply to each column.